install.sh

```diff
@@ -77,6 +77,16 @@ update_path() {
     fi
 }
 
+remove_existing_files() {
+    local existing_files=(fastn_macos_x86_64 fastn_linux_musl_x86_64 fastn_controller_linux_musl_x86_64)
+
+    for file in "${existing_files[@]}"; do
+        if [ -f "$file" ]; then
+            rm -f "$file"
```

Comment:
Isn't `rm fastn_macos_x86_64 fastn_linux_musl_x86_64 fastn_controller_linux_musl_x86_64` enough?

Reply:
You are trying to ignore the error message?

Reply:
yes, maybe the installation failed because insufficient permissions, and the user runs the installer again

Reply:
thanks, changed it to: `rm -f fastn_macos_x86_64 fastn_linux_musl_x86_64 fastn_controller_linux_musl_x86_64`